manim/mobject/types/vectorized_mobject.py

```diff
@@ -969,20 +969,31 @@ def get_curve_functions_with_lengths(
             yield self.get_nth_curve_function_with_length(n)
 
     def point_from_proportion(self, alpha: float) -> np.ndarray:
-        """Get the bezier curve evaluated at a position P,
-        where P is the point corresponding to the proportion defined by the given alpha.
+        """Gets the point at a proportion along the mobject.
```

Comment:
vmobject?

Reply:
And maybe:
Gets the point at a proportion along the path of the mobject.

Reply:
I felt weird using `vmobject` without capitalizing it to `VMobject` but then that felt weird because there's also `OpenGLVMobject` which is a wholly different class. I can change it to `vmobject` if you still think that's better though.

Also 'Gets the point at a proportion along the path of the mobject." is indeed better wording, thanks :)